### ISSUE TITLE
Failed to log in to travis-ci.org

### ISSUE THREAD

**Issue by GodfatherThe** [travis-ci/travis-ci#4877]
I'm trying to log in into travis-ci.org by pressing big green button "Sign in with GitHib" but seems it doesn't work - it writes "Signing in..." but nothing happens. I've waited for like a 10 minutes, but nothing changed. Also I've tried several browsers such as IE, Firefox and Chrome - no difference. Any advice?

**Comment by mephir** [comment]
:+1: doesnt work with opera

**Comment by Bosek** [comment]
I had the same problem. I noticed something is blocking requesting some external pages(GET https://third-party-cookies.herokuapp.com/check net::ERR_BLOCKED_BY_CLIENT). Had firewall off, no adblock.

It seems Avira antivirus caused the problem. If you are using it, try "Unblock trackers" on travis-ci.org page.

**Comment by mephir** [comment]
@Bosek I used to use ESET which I disabled. I noticed some ssl errors. My colleague is facing the same problems.

**Comment by BanzaiMan** [comment]
Sorry for overlooking this. If this is still a problem, please open a new issue. The causes may have changed, and we would probably need a brand new investigation.
